CodeBenders/Assets/Scripts/ControlButton.cs

```diff
@@ -56,6 +46,10 @@ void TaskOnClick()
                     enemy.GetComponent<DragDrop>().enabled = false;
 
                 }
+                
+                // enable slingshot reload for player 1
```

Comment:
@bzhu36 @bharatht96 you might want to consult these two examples on how to disable slingshot firing for a specific player.